libraries/Microsoft.Bot.Builder.AI.QnA/Models/Constants.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.AI.QnA.Models
+{
+    /// <summary>
+    /// Constants used in QnA Maker Bot Builder and samples.
+    /// </summary>
+    public static class Constants
```

Comment:
Remove this file and move constants to other locations as discussed in code review meeting